### ISSUE TITLE
VoiceOver screenreader cannot click Godot buttons

### ISSUE THREAD

**Issue by PeterDrake** [godotengine/godot#120324]
### Tested versions

Reproducible in 4.6.3.stable

### System information

Godot v4.6.3.stable - macOS Sequoia (15.7.0) - Multi-window, 1 monitor - OpenGL 3 (Compatibility) - Apple M1 - Apple M1 (8 threads) - 16.00 GiB memory

### Issue description

The macOS VoiceOver screenreader _recognizes_ Godot buttons, and offers to click them, but they don't respond. VO makes a sound indicating that it thinks it has clicked the button. The button does respond to space or to a mouse click, so it's not broken.

### Steps to reproduce

Using the minimal reproduction project below, make and run a Mac build. Use VoiceOver to navigate to a button. (The navigation controls don't obey the structure within Godot, but that's a separate issue). Try using the VO command (ctrl-option-space) to click the button. The text on the button *should* change (and it does if you just hit space or click on the button), but it doesn't.

### Minimal reproduction project (MRP)

Use commit f5e074 from this repository:

https://github.com/PeterDrake/screenreader

**Comment by bruvzg** [comment]
Your project have only `button_down` signals connected, and screen reader `click` action is sending `pressed` (and `toggled` for toggle button) signals directly. `pressed` is a signal typically used for button click.

**Comment by bruvzg** [comment]
While `button_down` is a rarely used signal (for the reference Godot editor uses it 2 times vs. 840 for `pressed`), I do not see any reason not to send both.

**Comment by PeterDrake** [comment]
That solves the problem -- thanks!

Does this now become a documentation issue?
